client/src/components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect, useState } from "react";
+import { OutlinedInput, InputLabel, InputAdornment } from "@material-ui/core";
+import LocationOnIcon from "@material-ui/icons/LocationOn";
+import LocationOffIcon from "@material-ui/icons/LocationOff";
+import { isDebug } from "src/config";
+
+function SearchBar({
```

Comment:
Add some doc on the functionality of the searchBar

Reply:
added documentation comments in style of JSDocs, however, still need to configure our project to generate docs #32